src/main/webapp/reports/inventoryReports/grn_cash.xhtml

```diff
@@ -300,7 +300,7 @@
                                 </h:outputText>
                             </p:column>
                             <p:column headerText="Purchase Value" styleClass="text-end" headerStyleClass="text-end" width="11em">
-                                <h:outputText value="#{bi.billItemFinanceDetails.valueAtPurchaseRate}">
```

Comment:
This should work for new bills. Old bills need to have their data updated, as the calculation was done recently. If not working, we have to fix it at the point of GRN Creation.